lib/reek/context/visibility_tracker.rb

```diff
@@ -39,7 +39,7 @@ def set_child_visibility(child)
         child.visibility = tracked_visibility
       end
 
-      # return [Boolean] If the visibility is public or not.
+      # r@eturn [Boolean] If the visibility is public or not.
```

Comment:
Whoopsie! My daughter was crying, so I was pushing the last change in a hurry :laughing: